projects/canopy/src/lib/accordion/accordion-item/accordion-item.component.ts

```diff
@@ -45,6 +46,9 @@ export class LgAccordionItemComponent implements AfterContentInit, OnChanges, On
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   _contentTemplate: TemplateRef<any>;
 
+  @HostBinding('role') role = 'listitem';
+  // used for adding more context to the heading
+  @Input() ariaDescribedBy: string = null;
```

Comment:
Tip: unless you have dedicated implementation for using `null` and you just need a falsy value, there's no need to assign a value to the property because by default it will be undefined.